### ISSUE TITLE
Drawer is failing accessibility

### ISSUE THREAD

**Issue by silviyaboteva** [telerik/kendo-react#962]
During accessibility testing, there are 3 axe violations because of the assigned aria roles. The violations are:

1. aria-required-children: Required ARIA child role not present: option (on .k-dropdown-wrap)
2. aria-required-parent: Required ARIA parent role not present: listbox (on .k-drawer-item)
3. List element has direct children with a role that is not allowed: option (on .k-drawer-items)

**Comment by silviyaboteva** [comment]
Fixed in `v4.8.0-dev.202106181103`.
